ligo/stablecoin/actions.ligo

```diff
@@ -240,4 +333,236 @@ function is_operator_action
   } with
   ( list [is_operator (parameter, store.operators)]
   , store
+  )
+
+(*
+ * Pauses whole contract in order to prevent all transferring, burning,
+ * minting and allowance operations. All other operations remain
+ * unaffected. Fails if the contract is already paused.
+ *)
+function pause
+  ( const parameter : pause_params
+  ; const store     : storage
+  ) : entrypoint is block
+{ ensure_not_paused (store)
+; authorize_pauser (store)
+} with
+  ( (nil : list (operation))
+  , store with record [paused = True]
+  )
+
+(*
+ * Unpauses whole contract so that all transferring, burning, minting and
+ * allowance operations can be performed. Fails if the contract is already
+ * unpaused.
+ *)
+function unpause
+  ( const parameter : unpause_params
+  ; const store     : storage
+  ) : entrypoint is block
+{ ensure_is_paused (store)
+; authorize_pauser (store)
+} with
+  ( (nil : list (operation))
+  , store with record [paused = False]
+  )
+
+(*
+ * Adds minter to minting allowances map setting its allowance
+ * to 0. Resets the allowance if associated address is already
+ * present. Fails if sender is not master minter.
+ * TODO
+ *)
+function configure_minter
+  ( const parameter : configure_minter_params
+  ; const store     : storage
+  ) : entrypoint is block
+{ ensure_not_paused (store)
+; authorize_master_minter (store)
+; case parameter.1.0 of
+    None -> case store.minting_allowances[parameter.0] of
+      Some (u) -> failwith ("Current minting allowance is required for changing already set minting allowance for the given address")
+    | None -> skip
+    end
+  | Some (current_minting_allowance) ->
+      case store.minting_allowances[parameter.0] of
```

Comment:
Would it be beneficial to define a variable/constant holding `store.minting_allowances[parameter.0]`? It takes some non-trivial computation to obtain this value. If we define `cur_allowance = store.minting_allowances[parameter.0]` will it be computed once?